app/src/main/java/org/oppia/android/app/options/OptionsActivity.kt

```diff
@@ -23,8 +23,11 @@ import javax.inject.Inject
 
 private const val SELECTED_OPTIONS_TITLE_SAVED_KEY = "OptionsActivity.selected_options_title"
 private const val SELECTED_FRAGMENT_SAVED_KEY = "OptionsActivity.selected_fragment"
+/**Key used to identify [ReadingTextSizeFragment]*/
```

Comment:
Ditto here & everywhere else in the PR. Please go back and update all new KDocs to match the formatting suggestions from my earlier comments & edits in this PR review.